lib/task-data/schemas/sku-firmware-update.json

```diff
@@ -0,0 +1,24 @@
+{
+    "$schema": "rackhd-task-schema.json",
+    "copyright": "Copyright 2016, EMC, Inc.",
+    "title": "SKU Firmware Update",
+    "description": "The schema for firmware update tasks via SKU pack",
+    "describeJob": "Job.Linux.Commands",
+    "definitions": {
+        "file": {
+            "description": "Image file, must have specified suffix and be placed in sku pack static/bios, static/bios folers or RackHD default file repo ",
```

Comment:
From the patter, the file can be either bios or bmc, but  the description is only for static/bios, please fix the description.